build/test.py

```diff
@@ -239,7 +267,7 @@ def __init__(self, description):
         help='Configure license servers for the custom asset playback test. '
              'May be specified multiple times to configure multiple key '
              'systems.',
-        type=str,
+        type=_KeyValidatorType('KEY_SYSTEM_ID'),
```

Comment:
There is no specific expected key, and "KEY_SYSTEM_ID" is not a valid key.  KEY_SYSTEM_ID is a placeholder to tell people that a key system ID must be provided, such as "com.widevine.alpha" or "com.microsoft.playready" (actual examples).

We shouldn't restrict to a specific key.  We should only require the key=value format.

Reply:
I see, just to double check my understanding, we just need to validate that the string we get from `--test-custom-license-server` is in `some_key=some_value` format.

Reply:
done!